hydromt_wflow/data/parameters_data.yml

```
@@ -86,6 +90,29 @@ vito_mapping_default:
     source_version: 1.0
   path: lulc/vito_mapping.csv
 
+glcnmo_mapping_default:
+  data_type: DataFrame
+  driver: csv
+  kwargs:
```

```suggestion
  driver_kwargs:
```
kwargs is now deprecated in favor of driver_kwargs in hydromt. Can you update for the rest as well?

---

Fixed it!